notebooks/patch_to_point/patch_to_point.ipynb

```diff
@@ -0,0 +1,689 @@
+       "0   terrascope  2016-10-30  2018-05-03   \n",
+       "\n",
+       "                                  s1_collection_path  \\\n",
+       "0  /data/users/Public/vincent.verelst/world_cerea...   \n",
+       "\n",
+       "                                  s2_collection_path   epsg  \\\n",
+       "0  /data/users/Public/vincent.verelst/world_cerea...  32736   \n",
+       "\n",
+       "                                            geometry  \n",
+       "0  [[34.453280157301236, -0.9129803702473933, 34....  "
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import pystac\n",
+    "from typing import List\n",
+    "from openeo_gfmap import Backend\n",
+    "\n",
+    "def create_job_dataframe(backend: Backend, s1_collection_paths: List[str], s2_collection_paths: List[str]) -> pd.DataFrame:\n",
+    "    columns = ['backend_name', 'start_date', 'end_date', 's1_collection_path', 's2_collection_path', 'epsg', 'geometry']\n",
+    "    rows = []\n",
+    "    for idx, collection_path in enumerate(s1_collection_paths):\n",
+    "        coll = pystac.read_file(collection_path)\n",
+    "        start_date = coll.extent.temporal.intervals[0][0]\n",
+    "        end_date = coll.extent.temporal.intervals[0][1]\n",
+    "        geometry = coll.extent.spatial.bboxes\n",
+    "        epsg = collection_path.split('/')[-2][11:]\n",
+    "\n",
+    "        rows.append(\n",
+    "            pd.Series(\n",
+    "                dict(zip(columns, [backend.value, start_date.strftime('%Y-%m-%d'), end_date.strftime('%Y-%m-%d'), collection_path, s2_collection_paths[idx], epsg, geometry]))\n",
+    "            )\n",
+    "        )\n",
+    "\n",
+    "    return pd.DataFrame(rows)\n",
+    "\n",
+    "job_df = create_job_dataframe(Backend.TERRASCOPE, s1_collection_paths, s2_collection_paths)\n",
+    "\n",
+    "job_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### RDM API Interaction\n",
+    "\n",
+    "This is a placeholder to get a list of the URL to all geoparquets in the RDM intersection with the S1 and S2 STAC collections.  \n",
+    "We can use `query_ground_truth` to create a GeoParquet file to a specific output folder with all points in the RDM that are within a specified Polygon. The Polygon will be constructed from the job_df geometry."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create_datacube"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, create the `create_datacube` function to be given to the GFMapJobManager is created. For this we cannot use worldcereal_classification's `worldcereal_preprocessed_inputs` in it's current form. Maybe best to discuss how we unify as much as possible the preprocessing of the different scripts.\n",
```

Comment:
This is indeed a key point! We need to keep these as aligned as much as possible.

Reply:
Can we add this to the agenda to discuss in our meeting on monday?

Reply:
yes!